hacks_mbof/settings.py

```diff
@@ -112,12 +112,13 @@
         'PASSWORD': getenv('DJANGO_DB_PASSWORD', ''),
         'HOST': getenv('DJANGO_DB_HOST', ''),
         'PORT': getenv('DJANGO_DB_PORT', ''),
-        'OPTIONS': {
-            'charset': 'utf8mb4',
-        },
     }
 }
 
+if DATABASES['default']['ENGINE'] == 'django.db.backends.mysql':
+    charset = {'charset': 'utf8mb4'}
+    DATABASES['default']['OPTIONS'] = charset
```

Comment:
Why not combine these two lines into one: `DATABASES['default']['OPTIONS'] = {'charset': 'utf8mb4'}`

The documentation, where it explains how to configure this app to use a MySQL DB, it should explain why the charset option needs to be set and which value should be used.

Thinking about this more, setting the charset like this doesn't look like the right thing to do.  What if there's a future charset type that would work, but is not `utf8mb4`?  This code would replace it, which can be confusing.

Rather than this conditional setting of charset, it would be better to add a check in the app that writes some utf8mb4-specific string to the DB (catching exceptions), then attempts to read the value back (again catching exceptions).  If there are exceptions or the strings written to and read back from the DB don't match, then report an error and die.